api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -270,47 +270,25 @@ public static String getFileAsString(File file) throws IOException {
 	 * @param inputStream Stream to copy from
 	 * @param outputStream Stream/location to copy to
 	 * @throws IOException thrown if an error occurs during read/write
+	 * @should not copy the outputstream when outputstream is null
+	 * @should not copy the outputstream if inputstream is null
+	 * @should copy inputstream to outputstream
```

Comment:
you should also test that the outputstream is closed at completion.

so "should copy inputstream to outputstream and close the outputstream"